vm-setup/roles/v1aX_integration_test/templates/controlplane_ubuntu.yaml

```diff
@@ -81,17 +87,16 @@ spec:
                   192.168.111.249
               }
           }
-      - path: /etc/netplan/50-cloud-init.yaml
+      - path: /etc/network/interfaces
         owner: root:root
         permissions: '0644'
         content: |
-          network:
-              ethernets:
-                  enp1s0:
-                      dhcp4: true
-                  enp2s0:
-                      dhcp4: true
-              version: 2
+          # interfaces(5) file used by ifup(8) and ifdown(8)
+          auto lo
+          iface lo inet loopback
+          ## To configure a dynamic IP address
+          auto enp2s0
+          iface enp2s0 inet dhcp
```

Comment:
that change is not correct, it should be netplan in use.

Reply:
done